examples/duet/vae/AutoEncoder_Syft_Data_Scientist.ipynb

```diff
@@ -1,8 +1,395 @@
 {
```

Comment:
<p>Change loopback to the "xxx" string and remove the logging line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5299/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done